src/styles/layout/sidebar/styles.less

```diff
@@ -72,29 +79,34 @@
         margin-right: 12px;
       }
 
+      // TODO: Apply a class to these elements.
```

Comment:
Let's please always create issues for these todos, as it otherwise will never happen.